src/Auth0.php

```diff
@@ -108,6 +109,24 @@ public function login(
         header('Location: ' . $this->authentication()->getLoginLink($params));
     }
 
+    /**
```

Comment:
Unrelated to this PR, but this suggestion came through via the community forums and I thought it was a good one, so I'm sneaking it in here.